cds_ils/importer/providers/ebl/rules/document.py

```diff
@@ -125,6 +126,14 @@ def identifiers(self, key, value):
         }
         if isbn not in _identifiers:
             _identifiers.append(isbn)
+    if "q" in value and self.get("_eitem", {}).get("_type", None) == "audiobook":
```

Comment:
this part is a bit unreadable, how about extracting
```
_eitem_type = _eitem.get("_type", "e-book")
```
on the top of the method and then use the variable here ?